texar/torch/modules/connectors/connectors.py

```diff
@@ -339,11 +322,10 @@ def forward(self,  # type: ignore
         Args:
             inputs: The input (structure of) tensor to pass forward.
 
-        :returns:
+        Returns:
```

Comment:
Using `:returns:` here is necessary because we have colons (`:`) in the paragraph. The `sphinx-napoleon` package would incorrectly interpret this as annotating a return type. Same for a bunch of docstrings below.

Reply:
Fixed in 14ee0df